examples/hello_server.cc

```diff
@@ -1,33 +1,38 @@
-/* 
+/*
    Mathieu Stefani, 13 février 2016
-   
+
    Example of an hello world server
 */
 
+#include <pistache/endpoint.h>
 
-#include "pistache/endpoint.h"
-
-using namespace Pistache;
-
-class HelloHandler : public Http::Handler {
+class HelloHandler : public Pistache::Http::Handler {
 public:
 
     HTTP_PROTOTYPE(HelloHandler)
 
-    void onRequest(const Http::Request& request, Http::ResponseWriter response) {
```

Comment:
Please do not reformat the code when making changes unrelated to formatting.

Its fine that you "(void)request;", but there was no need to split line 17 into 4 lines. 

I am heavily inclined to NACK your PR as-is, as it introduces many unnecessary diffs.

Reply:
Sorry about that. The initial PR was a bit cleaner and I may have been a bit careless during this second "update". Give me a few minutes and I'll fix the worst parts.